src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -4407,6 +4415,28 @@ theorem udiv_twoPow_eq_of_lt {w : Nat} {x : BitVec w} {k : Nat} (hk : k < w) : x
   have : 2^k < 2^w := Nat.pow_lt_pow_of_lt (by decide) hk
   simp [bitvec_to_nat, Nat.shiftRight_eq_div_pow, Nat.mod_eq_of_lt this]
 
+theorem toInt_mul_toInt_lt {x y : BitVec w} : x.toInt * y.toInt ≤ 2 ^ (w * 2 - 2) := by
```

Comment:
```suggestion
theorem toInt_mul_toInt_lt {x y : BitVec w} : x.toInt * y.toInt < 2 ^ (w * 2 - 2) := by
```

Reply:
Why was this closed @luisacicolini. It would be useful if you can always add a comment with respect to how you resolved this comment.